integrations/gtag/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-integration-gtag",
   "description": "The Gtag analytics.js integration.",
-  "version": "1.0.0-beta.1",
+  "version": "1.0.1-beta.1",
```

```suggestion
  "version": "1.0.1",
```

I don't think beta tag is necessary in a.js repo and it just makes it a bit confusing.